portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +25,21 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
-  @Override
+    private List<String> fact;
+    
+    @Override
+    public void init() {
+        fact = new ArrayList<>();
+        fact.add("I was originally going to be an Electrical Engineer but CS is much more fun!!!");
+        fact.add("Got 2 dogs, A Chihuahua and a Shih Tzu name Clyde and Coco");
```

Comment:
can "fact" be a const instead?

Reply:
changed fact to final static to make it compile-time constant